docs/UsersGuide/source/BackgroundInfo/CCPPUpdates.rst

```diff
@@ -0,0 +1,95 @@
+:orphan:
```

Comment:
For consistency, it would be nice to have all the bullets start with a verb (e.g. "Added ...") or noun (e.g. "Option for..."), but you probably got this from different people and put it together, so it's just a recommendation.

Reply:
Agreed... and I was trying to do that for v2.2.0 but then decided not to mess with it because I was afraid I'd distort the meaning (got it from the CCPP folks, and apparently, there was a LOT of back-and-forth on things, so I decided to leave well enough alone).